doc/mediasdk_release_notes.md

```diff
@@ -205,8 +191,9 @@ This release is subject to the following known limitations:
     - Multi Frame Encode (MFE), HEVC Flexible Encode Infrastructure only supported on Skylake
     - VP9 decoder is supported starting from Kabylake platform
     - VP9 encoder is supported starting from Icelake platform
-    - Support for Tigerlake, Elkhartlake and Jasperlake platforms is preliminary and may not be fully functional
     - SW fallback is unsupported for all components but MJPEG
+    - Keem Bay require a separate runtime library
+    - Some Keem Bay features are specific to this platform and not available on GEN graphics 
```

Comment:
This one is not clear. Which features are specific to KeemBay? Why we talk about KeemBay features, but not API not supported on GEN graphics?